src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,40 +1,67 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { Todo } from '../../types/Todo';
+import { actions } from '../../features/currentTodo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
 
 export const TodoModal: React.FC = () => {
+  const dispach = useAppDispatch();
```

Comment:
There is a typo in 'dispatch'. It should be 'dispatch' instead of 'dispach'.

Reply:
done